gazebo_ros_control/pid_gains:
  leg1coxa: {p: 100.0, i: 0.01, d: 10.0}
  leg1femur: {p: 100.0, i: 0.01, d: 10.0}
  leg1tibia: {p: 100.0, i: 0.01, d: 10.0}
  leg2coxa: {p: 100.0, i: 0.01, d: 10.0}
  leg2femur: {p: 100.0, i: 0.01, d: 10.0}
  leg2tibia: {p: 100.0, i: 0.01, d: 10.0}
  leg3coxa: {p: 100.0, i: 0.01, d: 10.0}
  leg3femur: {p: 100.0, i: 0.01, d: 10.0}
  leg3tibia: {p: 100.0, i: 0.01, d: 10.0}
  leg4coxa: {p: 100.0, i: 0.01, d: 10.0}
  leg4femur: {p: 100.0, i: 0.01, d: 10.0}
  leg4tibia: {p: 100.0, i: 0.01, d: 10.0}
  leg5coxa: {p: 100.0, i: 0.01, d: 10.0}
  leg5femur: {p: 100.0, i: 0.01, d: 10.0}
  leg5tibia: {p: 100.0, i: 0.01, d: 10.0}
  leg6coxa: {p: 100.0, i: 0.01, d: 10.0}
  leg6femur: {p: 100.0, i: 0.01, d: 10.0}
  leg6tibia: {p: 100.0, i: 0.01, d: 10.0}
  tail_joint: {p: 100.0, i: 0.01, d: 10.0}
